native/cocos/platform/apple/modules/CanvasRenderingContext2DDelegate.mm

```diff
@@ -72,8 +72,8 @@ @interface CanvasRenderingContext2DDelegateImpl : NSObject {
     cc::Data        _imageData;
     NSBezierPath *  _path;
 
-    CanvasTextAlign      _textAlign;
-    CanvasTextBaseline   _textBaseLine;
+    cc::ICanvasRenderingContext2D::TextAlign      _textAlign;
+    cc::ICanvasRenderingContext2D::TextBaseline   _textBaseLine;
```

Comment:
Can use TextAlign and TextBaseline instead?

Reply:
Is there any way in Object-C? I don't quite understand. I can't use 'using' here

Reply:
Yep, can not.